docs/Components.md

```diff
@@ -0,0 +1,98 @@
+# Components
+
+# Card
```

Comment:
What is the ordering for this components file? Some decisions like `EditableArrayCard` coming before `ArrayCard` are confusing

Reply:
Yeah the ideal was that it was simple=>complex, more used => less used. Fixed that dependency ordering problem.